float32 cs0  # in degrees; horizontal:  c/stbd  vertical: -b/top
float32 cs1  # in degrees; horizontal: -e/port  vertical: d/bottom
